funfact/loss.py

```diff
@@ -45,7 +45,7 @@ def __call__(self, model, target, reduction='mean', sum_vec=True,
 class MSE(Loss):
 
     def _loss(self, model, target):
-        return ab.square(ab.subtract(model, target))
+        return ab.square(ab.abs(ab.subtract(model, target)))
```

Comment:
Maybe leave a comment to say that `abs` is for handling complex data? Otherwise, it may appear as redundant given the `square` operation.

Reply:
Done.